### ISSUE TITLE
Training loss reported is an EMA with a (too?) high beta value

### ISSUE THREAD

**Issue by sebderhy** [fastai/fastai#4063]
## Issue Description
The training loss reported by fastai in its output table when fitting a learner seems significantly higher than both the validation loss and the actual raw training losses, even with very simple models where training and valid losses should be more or less equal. This seems to be because the training loss reported is `AvgSmoothLoss` with a high beta value (0.98), while validation losses are reported without smoothing.

I think it may be a bit misleading to see a training loss much higher than what it actually is at the end of the epoch. For example, it caused me to think that I had a bug in my model (I thought it had some hidden dropout somewhere). I understand why we want to average, but why not averaging over the last batches? 

## Minimal Reproduction Code
```python
import torch
import torch.nn as nn
import torch.nn.functional as F
from torch.utils.data import Dataset, DataLoader
from fastai.learner import Learner
from functools import partial
from fastai.optimizer import OptimWrapper
from fastai.data.core import DataLoaders

class SimpleDataset(Dataset):
    def __init__(self, num_samples=10_000):
        self.num_samples = num_samples
        
    def __len__(self):
        return self.num_samples
    
    def __getitem__(self, idx):
        x = torch.rand(1)  # Random value between 0 and 1
        return x, x  # Same value for input and target

class SimpleMLP(nn.Module):
    def __init__(self):
        super().__init__()
        self.net = nn.Sequential(
            nn.Linear(1, 64),
            nn.ReLU(),
            nn.Linear(64, 1)
        )
    
    def forward(self, x):
        return self.net(x)

train_ds = SimpleDataset(10_000)
valid_ds = SimpleDataset(1_000)
train_dl = DataLoader(train_ds, batch_size=1000, shuffle=True)
valid_dl = DataLoader(valid_ds, batch_size=1000)
dls = DataLoaders(train_dl, valid_dl)

model = SimpleMLP()
learn = Learner(dls, model, loss_func=F.l1_loss, opt_func=partial(OptimWrapper, opt=torch.optim.Adam))
learn.fit_one_cycle(n_epoch=10, lr_max=1e-3)
```

## Current Output
```
epoch	train_loss	valid_loss	time
0	0.220724	0.219236	00:00
1	0.200737	0.147620	00:00
2	0.168200	0.089695	00:00
3	0.137362	0.053103	00:00
4	0.107697	0.015643	00:00
5	0.082520	0.010221	00:00
6	0.065091	0.009564	00:00
7	0.052148	0.007720	00:00
8	0.042441	0.007580	00:00
9	0.035105	0.007896	00:00
```

## Analysis of the Issue
The discrepancy seems to be due to `AvgSmoothLoss` using exponential moving average with beta=0.98. Here's a callback showing the actual raw losses vs the reported EMA:

```python
from fastai.callback.core import Callback
model = SimpleMLP()

class LossAnalysisCallback(Callback):
    def __init__(self, beta=0.98):
        super().__init__()
        self.beta = beta
        self.ema_loss = None
        self.raw_losses = []
        
    def after_batch(self):
        if self.training:
            current_loss = self.learn.loss.item()
            if self.ema_loss is None:
                self.ema_loss = current_loss
            else:
                self.ema_loss = self.beta * self.ema_loss + (1-self.beta) * current_loss
            print(f"Raw loss: {current_loss:.6f}, EMA: {self.ema_loss:.6f}")

# Add callback to see raw vs EMA losses
learn = Learner(dls, model, loss_func=F.l1_loss, 
                opt_func=partial(OptimWrapper, opt=torch.optim.Adam),
                cbs=[LossAnalysisCallback()])

learn.fit_one_cycle(n_epoch=3, lr_max=1e-3)
```

Losses displayed:
```
epoch	train_loss	valid_loss	time
0	0.432030	0.386449	00:00
1	0.369593	0.286083	00:00
2	0.328722	0.265476	00:00
```

Actual losses:
```
Raw loss: 0.472060, EMA: 0.472060
Raw loss: 0.468120, EMA: 0.471981
Raw loss: 0.453600, EMA: 0.471613
Raw loss: 0.452861, EMA: 0.471238
Raw loss: 0.441241, EMA: 0.470638
Raw loss: 0.443718, EMA: 0.470100
Raw loss: 0.427866, EMA: 0.469255
Raw loss: 0.399145, EMA: 0.467853
Raw loss: 0.388143, EMA: 0.466259
Raw loss: 0.390184, EMA: 0.464737
Raw loss: 0.368422, EMA: 0.462811
Raw loss: 0.357878, EMA: 0.460712
Raw loss: 0.352365, EMA: 0.458546
Raw loss: 0.332993, EMA: 0.456034
Raw loss: 0.317996, EMA: 0.453274
Raw loss: 0.318360, EMA: 0.450575
Raw loss: 0.303432, EMA: 0.447633
Raw loss: 0.289474, EMA: 0.444469
Raw loss: 0.291538, EMA: 0.441411
Raw loss: 0.270920, EMA: 0.438001
Raw loss: 0.273341, EMA: 0.434708
Raw loss: 0.274699, EMA: 0.431508
Raw loss: 0.280689, EMA: 0.428491
Raw loss: 0.271261, EMA: 0.425347
Raw loss: 0.268849, EMA: 0.422217
Raw loss: 0.265930, EMA: 0.419091
Raw loss: 0.266798, EMA: 0.416045
Raw loss: 0.257287, EMA: 0.412870
Raw loss: 0.261781, EMA: 0.409848
Raw loss: 0.262985, EMA: 0.406911
```

The raw training losses are much lower than the EMA loss (almost x2 at the end!).

## Environment
- fastai version: 2.7.18 (latest)
- PyTorch version: 2.5.1
- Python version: 3.10.0
